src/main/java/net/mcreator/generator/Generator.java

```diff
@@ -280,6 +280,18 @@ public boolean generateElement(GeneratableElement element) {
 		return new ArrayList<>(generatorFiles);
 	}
 
+	public List<String> getElementLocalizationKeys(GeneratableElement element) {

```

Comment:
This should be in LocalizationUtils

Reply:
It is there. This method is a shortcut for that method needed to get definition map from generator configuration and generator object itself.

Reply:
Why does single line LocalizationUtils.getLocalizationKeys then exist, if it is only used by this method? Arraylist can probably also be created here inline